src/base58.c

```diff
@@ -44,8 +45,9 @@ int btc_base58_decode(void* bin, size_t* binszp, const char* b58)
     size_t binsz = *binszp;
     const unsigned char* b58u = (const void*)b58;
     unsigned char* binu = bin;
-    size_t outisz = (binsz + 3) / 4;
-    uint32_t outi[outisz];
+    const size_t outisz = (binsz + 3) / 4;
+    const size_t outilen = sizeof(uint32_t) * outisz;
+    uint32_t *outi = (uint32_t *)alloca(outilen);
```

Comment:
Is there a reason to heap allocate those buffers? Won't this compile on Visual Studio as it is?
`alloca` seems to be the wrong choice.
From the man pages:
```
    alloca() is machine and compiler dependent; its use is discouraged.
```

Heap allocation should always go over the memory mapper (use `btc_malloc()`, `btc_free()`).